mgmake/generate.go

```diff
@@ -20,24 +21,33 @@ type templateTargets struct {
 
 // GenerateMakefile is a mage target that ...
 func GenerateMakefile(makefile string) error {
+	genDir := filepath.Dir(makefile)
+	mgTargets := filepath.Join(genDir, "mg_targets.mk")
```

Comment:
prefer kebab-case in file names over snake_case

Reply:
:+1: